src/main/java/org/terasology/structureTemplates/internal/events/RequestStructureTemplatePropertiesChange.java

```diff
@@ -49,4 +52,6 @@ public Prefab getPrefab() {
     public Integer getSpawnChance() {
         return spawnChance;
     }
+
+    public AnimationType getAnimationType() { return animationType; }
```

Comment:
Generally even when a method can be a one liner like this it tends to still get three lines, no big deal though :-)